pages/glossary/included-in-the-accessibility-tree.md

```diff
@@ -11,7 +11,7 @@ Elements included in the accessibility tree of platform specific accessibility A
 
 The general rules for when elements are included in the accessibility tree are defined in the [core accessibility API mappings](https://www.w3.org/TR/core-aam/). For native markup languages, such as HTML and SVG, additional rules for when elements are included in the accessibility tree can be found in the [HTML accessibility API mappings (working draft)](https://www.w3.org/TR/html-aam/) and the [SVG accessibility API mappings (working draft)](https://www.w3.org/TR/svg-aam/).
 
-For more details, see [examples of included in the accessibility tree][].
+For more details, see [examples of included in the accessibility tree](https://act-rules.github.io/pages/examples/included-in-the-accessibility-tree/).
+
+Elements with `aria-hidden="true"` should not be included in the accessibility tree. However, some browsers include [focusable](#focusable) elements with `aria-hidden="true"`in the accessibility tree. In such a browser, a user of assistive technology could still interact with the element by using sequential keyboard navigation, even though the element should not be included in the accessibility tree.
```

Comment:
Maybe like this?

```suggestion
Some browser will leave [focusable][] elements with a [hidden state][] of `true` in the accessibility tree. Because they are hidden, these elements are considered **not** included in the accessibility tree.

**Note:** Users of assistive technologies might still be able to interact with elements that are not included in the accessibility tree. An example of this is a [focusable][] element with an `aria-hidden` attribute with a value of `true`. Such an element could still be interacted using sequential keyboard navigation regardless of the assistive technologies used, even though the element would not be included in the accessibility tree.

For more details, see [examples of included in the accessibility tree][].

[focusable]: #focusable
[hidden-state]: #hidden-state
[examples of included in the accessibility tree]: https://act-rules.github.io/pages/examples/included-in-the-accessibility-tree/
```